### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by ItsSitanshu**
Update grammar for better communication of information about the project

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -262,11 +262,13 @@ Ccache can be enabled by configuring with `-DUSE_CCACHE=<ON | OFF>`.
 * [ ] Add more algorithms  
 * [ ] Add support for nonlinear systems automatic differentiation, so that the Jacobians are automatically computed (see [autodiff](https://autodiff.github.io/tutorials/))  
 * [ ] Add a README.md to each example folder, to explain the theory  
-* [ ] Cache the packages downloaded by CPM.CMake (currently everything is re-downloaded everytime a new example is built)
+* [ ] Cache the packages downloaded by CPM.CMake (currently everything is re-downloaded every timesuggestions a new example is built)
```
Comment (giacomo-b) [inline README.md:265]: every timesuggestions?

File: README.md [hunk 2]
```diff
@@ -215,7 +215,7 @@ cmake --build build/doc --target GenerateDocs
 open build/doc/doxygen/html/index.html
 ```
 
-To build the documentation locally, you will need _Doxygen_, _jinja2_ and _Pygments_ installed your system.
+To build the documentation locally, you will need _Doxygen_, _jinja2_ and _Pygments_ installed in your system.
```
Comment (giacomo-b) [inline README.md:218]: Should be "on"

File: README.md [hunk 3]
```diff
@@ -14,7 +14,7 @@ A header-only, fully-templated C++ library for robotics algorithms.
 
 This repo is inspired by [AtsushiSakai/PythonRobotics](https://github.com/AtsushiSakai/PythonRobotics). Instead of being just educational, this repo focuses on runtime speed and modularity. The `examples` folder contains short tutorials for each implemented algorithm.
 
-While this is still a work in progress, I would really appreciate it if you left any suggestion or just a star. Any help is greatly appreciated!
+While this is still a work in progress, I would set appreciate it if you left any suggestion or just a star. Any help is greatly appreciated!
```
Comment (giacomo-b) [inline README.md:17]: Did you want to remove "set"?

**Comment by ItsSitanshu** [comment]
CLOSING PR
____________

**Comment by giacomo-b** [comment]
@ItsSitanshu thank you for your contribution! I added a few quick comments, if you could fix that I would really appreciate it. Thanks!

File: README.md (hunk 2 above)
Comment (ItsSitanshu) [reply README.md:218]: Sorry m8

**Pull request closed without merging**